ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCUI-nonprod.json

```diff
@@ -696,5 +696,26 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "UPDATE_CASE_STATE_CASE_SETTLED",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "RES-SOL-ONE-SPEC-PROFILE",
+          "RES-SOL-TWO-SPEC-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE"
```

Comment:
it is not the defendant who changes the claim to settled. but the claimant. should be a different access profile. also not for the respondent solicitors. this is lip v lip so should be just CITIZEN-CLAIMAINT-PROFILE